docs/user-guide/creating-application/workflow/ci-pipeline.md

```diff
@@ -210,18 +210,25 @@ If one code is shared across multiple applications, `Linked Build Pipeline` can
 2. On the **App Configuration** page, select **Workflow Editor**.
 3. Select **+ New Workflow**.
 4. Select **Linked Build Pipeline**.
-5. Enter the following fields on the **Create linked build pipeline** screen:
+5. On the **Create linked build pipeline** screen:
+    * Search for the application in which the source CI pipeline is present.
+    * Select the source CI pipeline from the application that you selected above.
+    * Enter a new name for the linked CI pipeline. It cannot be the same as the source CI pipeline.
```

Comment:
`It cannot be the same as the source CI pipeline.` It has similar validation as other pipelines or something different?

Reply:
Okay. Removing that line.